com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree7Billboard.shader

```diff
@@ -33,6 +33,7 @@ Shader "Universal Render Pipeline/Nature/SpeedTree7 Billboard"
             #pragma vertex SpeedTree7Vert
             #pragma fragment SpeedTree7Frag
 
+            #pragma multi_compile _ LIGHTMAP_ON
```

Comment:
Lightmapping a billboard doesn't make sense, remove this.

Reply:
Right, that should have been removed.